### PR TITLE
implement "--localfile_only_world_readable" command line switch

### PR DESCRIPTION
**Pull request by MarkusTeufelberger**
see #743

### CODE REVIEW

File: nbviewer/app.py [hunk 1]
```diff
@@ -281,6 +282,7 @@ def init_options():
     define("debug", default=False, help="run in debug mode", type=bool)
     define("no_cache", default=False, help="Do not cache results", type=bool)
     define("localfiles", default="", help="Allow to serve local files under /localfile/* this can be a security risk", type=str)
+    define("localfile_only_world_readable", default=False, help="Only serve files that are readable by 'Other' on the local file system", type=bool)
```
Comment (parente) [inline nbviewer/app.py]: In the interest of having the most secure defaults and options to loosen them up, maybe this should be `--localfile_any_user` or some similarly named inverse?

Comment (MarkusTeufelberger) [reply nbviewer/app.py]: Nice suggestion, I'm not great at naming stuff. ;-)

Will do.

Comment (parente) [reply nbviewer/app.py]: No worries. Naming *is* hard.

**Comment by parente** [comment]
I'm sorry for the delay in returning to this PR. (4th quarter busy-ness.) This looks good to me. Thanks and merging!

**Pull request merged**